src/libraries/System.Transactions.Local/src/System/Transactions/TransactionScope.cs

```diff
@@ -806,7 +806,9 @@ private void InternalDispose()
                         // If we are supposed to commit on dispose, cast to CommittableTransaction and commit it.
                         if (null != _committableTransaction)
                         {
+#pragma warning disable CA1416 // Validate platform compatibility, not sure if we need to add condition instead
```

Comment:
This needs better fix

Reply:
Added [Unsupported on browser attributes](https://github.com/dotnet/runtime/pull/43363/files/422be5be362308a5c586faef93b75cb17a8c79ec#r510589841) for the enum values needed for initializing `_committableTransaction`, now its safe to suppress warning here

Reply:
I think we should mark whole TransactionScope as unsupported on Browser. It'd be a better user experience than this hit and miss approach.

Reply:
OK, i thought it will give a min functionality for the user.